draft-ietf-quic-recovery.md

```diff
@@ -576,7 +580,7 @@ discarded, the PTO and loss detection timers MUST be reset, because discarding
 keys indicates forward progress and the loss detection timer might have been set
 for a now discarded packet number space.
 
-#### Before Address Validation
+#### Before Server Address Validation
```

Comment:
Client?

Reply:
It's before the Client is sure the Server has validated the client's address, but I'll remove this change since it's unrelated.